contracts/governance/Vesting/VestingRegistryLogic.sol

```diff
@@ -289,6 +311,49 @@ contract VestingRegistryLogic is VestingRegistryStorage {
         return vestings[uid].vestingAddress;
     }
 
+    /**
+     * @dev check if the specific vesting address is team vesting or not
+     * @dev read the vestingType from vestingCreationAndTypes storage
+     *
+     * @param _vestingAddress address of vesting contract
+     *
+     * @return true for teamVesting, false for normal vesting
+     */
+    function isTeamVesting(address _vestingAddress) external view returns (bool) {
+        if (
+            !vestingCreationAndTypes[_vestingAddress].isSet ||
+            vestingCreationAndTypes[_vestingAddress].vestingType !=
+            uint256(VestingType.TeamVesting)
+        ) return false;
+        else return true;
```

Comment:
```suggestion
        return (
            vestingCreationAndTypes[_vestingAddress].isSet && 
            vestingCreationAndTypes[_vestingAddress].vestingType == 
            uint256(VestingType.TeamVesting)
        );
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)